src/init/inst-functions.sh

```diff
@@ -831,6 +831,7 @@ InstallCommon()
   ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} restart-wazuh ${PREFIX}/active-response/bin/
   ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} route-null ${PREFIX}/active-response/bin/
   ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} kaspersky ${PREFIX}/active-response/bin/
+  ${INSTALL} -m 0750 -o root -g ${OSSEC_GROUP} wazuh-slack ${PREFIX}/active-response/bin/
```

Comment:
Remember to also delete the `ossec-slack.sh` and `ossec-tweeter.sh` AR scripts in the `wazuh.sh` upgrade script.